src/main/views/still-working.njk

```diff
@@ -0,0 +1,30 @@
+{% extends "template.njk" %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% block beforeContent %}
+  {{ super() }}
+  {{ govukBackLink({
+  text: back,
+  href: "/"
+}) }}
+{% endblock %}
+
+{% block content %}
+	 {{ 
+    govukErrorSummary({
+      titleText: errorSummaryTitle,
+      errorList: getErrors(form.fields)
+    }) if getErrors(form.fields).length > 0
+  }}
```

Comment:
Please use error-summary component.

Reply:
updated to use the error summary component